README.md

```diff
@@ -1,5 +1,17 @@
 # Lumina
 
+<a href="https://github.com/eigerco/lumina/actions/workflows/ci.yml?query=branch%3Amain"><img src="https://github.com/eigerco/lumina/actions/workflows/ci.yml/badge.svg?branch=main" alt="Build Status" /></a>
```

Comment:
I'm wondering if they wouldn't look better if centered

Reply:
![image](https://github.com/user-attachments/assets/8757de4e-10dc-48c5-bcdf-6efce26884bc)
vs
![image](https://github.com/user-attachments/assets/c188f7c7-9ef0-4263-b225-b9b0b57fb88f)
vs
![image](https://github.com/user-attachments/assets/00af7c3e-084a-4081-ad1b-03ef6a1519a0)

I think I like the fully centered (last one) best.

Reply:
Agreed